drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/test/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/rightpanel/coverage/CoverageReportPresenterTest.java

```diff
@@ -38,7 +38,7 @@
 import org.uberfire.mvp.Command;
 
 import static org.drools.scenariosimulation.api.model.ScenarioSimulationModel.Type;
-import static org.jgroups.util.Util.assertEquals;
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 import static org.mockito.Matchers.anyString;
```

Comment:
Could we add to the populateCoverageReport test a case  if (simulationRunMetadata.getAvailable() == 0) 
Based on:
https://sonarcloud.io/component_measures?id=org.drools%3Adrools-wb&metric=new_branch_coverage&pullRequest=1347&selected=org.drools%3Adrools-wb%3Adrools-wb-screens%2Fdrools-wb-scenario-simulation-editor%2Fdrools-wb-scenario-simulation-editor-businesscentral-client%2Fsrc%2Fmain%2Fjava%2Forg%2Fdrools%2Fworkbench%2Fscreens%2Fscenariosimulation%2Fbusinesscentral%2Fclient%2Frightpanel%2Fcoverage%2FCoverageReportPresenter.java&view=list

As I can see you didnt change this code here , but it seams that this function is pretty good to cover

Reply:
@dupliaka Done